cmd/openshift-tests/openshift-tests.go

```diff
@@ -464,6 +466,7 @@ func mirrorToFile(opt *testginkgo.Options, fn func() error) error {
 func bindOptions(opt *runOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&opt.FromRepository, "from-repository", opt.FromRepository, "A container image repository to retrieve test images from.")
 	flags.StringVar(&opt.Provider, "provider", opt.Provider, "The cluster infrastructure provider. Will automatically default to the correct value.")
+	flags.BoolVar(&opt.Disconnected, "disconnected", opt.Disconnected, "Indicator that this cluster is disconnected")
```

Comment:
Unfortunately we can't change the signature of openshift-tests generally.  Disconnected state should be included in the provider struct.

Reply:
Sure, makes sense and I think that's a better solution overall.  Pushed an update, our tests would configure this for disconnected, instead of the flag:

```bash
TEST_PROVIDER='{"type": "baremetal","disconnected": true}'
```

Reply:
(Of course this only works if you specify the JSON, the previous case allows TEST_PROVIDER to just be a string for some platforms)